ci-operator/config/openshift/csi-operator/openshift-csi-operator-master.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+canonical_go_repository: github.com/openshift/csi-operator
+images:
+- from: base
+  inputs:
+    root:
+      as:
+      - '0'
+  to: csi-operator
```

Comment:
This name is too generic.  Please rename this to `storage-csi-operator` (or if there is an alternate naming that clearly groups the storage team operators I'm open to suggestions) in a PR and @ me